### ISSUE TITLE
.first not respecting ordering

### ISSUE THREAD

**Issue by sondrelg** [collerek/ormar#453]
Hi!

I'm trying to get the latest item from a table, wrt. a datetime-field, and I was a little surprised to learn how the `first` method works.

### What I expected

I've used Django a bit, so I was expecting .first() to get the first item in my queryset, with respect to existing constaints. I had already ordered the queryset the way
I wanted, so I was expecting this to work:

```python
latest = await MyModel.objects.order_by("-registered").first()
```

When that didn't work, I thought I might try to limit the size of the queryset myself, then use `.first()` as a shortcut over `.all()` since this will raise `NoMatch` etc. when the object doesn't exist:

```python
latest = await MyModel.objects.order_by("-registered").limit(1).first()
```

### Question

Is it correct that `.first()` disregards the queryset it's being called on and just returns the first item, sorted by pk? Why is this? Am I missing alternative syntax for a Django-style `.first()` method?

This is what I ended up with:

```python
latest = await MyModel.objects.order_by("-registered").limit(1).all()

if not latest:
    # execute the logic I'd otherwise have defined in the NoMatch case

return ...
```

Thanks!

_Originally posted by @sondrelg in https://github.com/collerek/ormar/discussions/427_

**Comment by collerek** [comment]
Should be fixed in 0.10.23 - please update and check

**Comment by sondrelg** [comment]
Can confirm that the query works when testing :) 

There is one potential issue (depending on your intentions around design): the .first() call throws a `NoMatch` when it cannot find a result. Django will return `None` in this case. Is this intentional @collerek?
